public_src/services/map.service.ts

```diff
@@ -653,7 +660,7 @@ export class MapService {
       },
     );
     if (this.highlight) {
-      this.highlight.addLayer(L.layerGroup([this.markerFrom, this.markerTo]));
```

Comment:
why L.layerGroup was deleted only there and not also few lines down? It looks like it works correctly also with L.layerGroup added here. I do not know what else could be problem again if you delete it here.

Reply:
As I understood it, this.highlight is supposed to be a layer group which contains all the layers for highlight (polyline and circle markers (from & to)). But when we are adding the layers to the this. highlight, initially (when this.highlight is null) , we assign it to be a layergroup with initial set of layer members as the first polyline (line 467). But when we add the second and the subsequent polylines to this layer group, we are adding a new  layergroup (with the second polyline) and then adding this layergroup to the main layergroup (this.highlight). I would not have changed it, but I was not able to iterate through this.highlight while adding and clearing text label.